lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = JSON.parse(
+	fsT.readFileSync('lib/js/importers/configs/allInOneConfig.json', 'utf8')
+);
+
+interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonSource: { data }
+) => {
+	let hexToCssVariable: Array<Object> = [];
```

Comment:
I suspect this type definition is wrong. In JS, when you use string keys, then array gets converted to `{}`

Reply:
BTW. you put `string` values, not `Object`

Reply:
Yes, its Object anyway.